pkg/neotest/signer.go

```diff
@@ -35,11 +36,20 @@ type SingleSigner interface {
 	Account() *wallet.Account
 }
 
+type MultiSigner interface {
+	Signer
+	// Single returns simple-signature signer for n-th account in list.
+	Single(n int) SingleSigner
```

Comment:
Unless we need to use `SingleSigner` received from this method as `Signer`, that'd be easier to do with the current interface scheme.